packages/app/src/systems/Core/components/Mnemonic/Mnemonic.tsx

```diff
@@ -39,7 +39,10 @@ export function Mnemonic({
   });
 
   async function handleCopy() {
```

Comment:
This could be written like that:

```ts
async function handleCopy() {
  const val = type === "read" ? initialValue : value
  return navigator.clipboard.writeText(val.join(' '));
}
```